docs/conf/portland/2021/news/cfp-opens.rst

```diff
@@ -19,7 +19,7 @@ We're looking for ideas and use-cases from all disciplines and roles, so whether
 
 You can read our full `Call for Proposals <https://www.writethedocs.org/conf/portland/{{year}}/cfp/>`__ on the website.
 
-The Call for Proposals will be open until **Midnight PST {{cfp.ends}}**.
```

Comment:
The timezone changes from PST to PDT during our news cycle 🤯

Reply:
Valid point, although what matters is the timezone it is in when the date occurs, right?

Reply:
Valid point, although what matters is the timezone it is in when the date occurs, right?